pkg/models/model_plan.go

```diff
@@ -32,15 +32,34 @@ type ModelCategory string
 
 // These constants represent the different values of ModelCategory
 const (
-	MCAccountableCare           ModelCategory = "ACCOUNTABLE_CARE"
-	MCDemonstration             ModelCategory = "DEMONSTRATION"
-	MCEBPaymentInitiatives      ModelCategory = "EPISODE_BASED_PAYMENT_INITIATIVES"
-	MCMedicaidAndChip           ModelCategory = "INIT_MEDICAID_CHIP_POP"
-	MCMedicareAndMedicaid       ModelCategory = "INIT__MEDICARE_MEDICAID_ENROLLEES"
-	MCAccelerateDevAndTest      ModelCategory = "INIT_ACCEL_DEV_AND_TEST"
-	MCSpeedBestPracticeAdopt    ModelCategory = "INIT_SPEED_ADOPT_BEST_PRACTICE"
-	MCPrimaryCareTransformation ModelCategory = "PRIMARY_CARE_TRANSFORMATION"
-	MCTBD                       ModelCategory = "UNKNOWN"
+	MCAccountableCare            ModelCategory = "ACCOUNTABLE_CARE"
+	MCDemonstration              ModelCategory = "DEMONSTRATION"
+	MCEBPaymentInitiatives       ModelCategory = "EPISODE_BASED_PAYMENT_INITIATIVES"
+	MCMedicaidAndChip            ModelCategory = "INIT_MEDICAID_CHIP_POP"
+	MCMedicareAndMedicaid        ModelCategory = "INIT__MEDICARE_MEDICAID_ENROLLEES"
+	MCAccelerateDevAndTest       ModelCategory = "INIT_ACCEL_DEV_AND_TEST"
+	MCSpeedBestPracticeAdopt     ModelCategory = "INIT_SPEED_ADOPT_BEST_PRACTICE"
+	MCPrimaryCareTransformation  ModelCategory = "PRIMARY_CARE_TRANSFORMATION"
+	MCTBD                        ModelCategory = "UNKNOWN"
+	MCDiseaseSpecificAndEpisodic ModelCategory = "DISEASE_SPECIFIC_AND_EPISODIC"
+	MCHealthPlan                 ModelCategory = "HEALTH_PLAN"
+	MCPrescriptionDrug           ModelCategory = "PRESCRIPTION_DRUG"
+	MCStateBased                 ModelCategory = "STATE_BASED"
+	MCStatutory                  ModelCategory = "STATUTORY"
+	MCToBeDetermined             ModelCategory = "TO_BE_DETERMINED"
+)
+
+// AdditionalModelCategory represents the additional category of a model
+type AdditionalModelCategory string
+
+// These constants represent the different values of AdditionalModelCategory
+const (
+	AMCAccountableCare            AdditionalModelCategory = "ACCOUNTABLE_CARE"
+	AMCDiseaseSpecificAndEpisodic AdditionalModelCategory = "DISEASE_SPECIFIC_AND_EPISODIC"
+	AMCHealthPlan                 AdditionalModelCategory = "HEALTH_PLAN"
+	AMCPrescriptionDrug           AdditionalModelCategory = "PRESCRIPTION_DRUG"
+	AMCStateBased                 AdditionalModelCategory = "STATE_BASED"
+	AMCStatutory                  AdditionalModelCategory = "STATUTORY"
```

Comment:
Don't need `AdditionalModelCategory` - can move these types to `ModelCategory` and ensure they align with SQL / GQL enums